### DISCUSSION TITLE
请问这种错位可能是什么原因导致的呀？

### DISCUSSION THREAD

**Discussion by Caikera** [nshen/learn-neovim-lua#129] [Q&A]
![disorder](https://github.com/Caikera/Questions/blob/main/disorder.png?raw=true)
GIF: https://github.com/Caikera/Questions/blob/main/disorder.gif

**Comment by nshen** [comment]
好神奇，是终端不行吧，mac 用iterm2 或 alacritty， windows 用 terminal

**Reply by Caikera** [reply]
我用的就是terminal，短暂得试了一下mobaXterm，暂时还没复现这个问题。
